esx_service/utils/auth_data.py

```diff
@@ -44,6 +47,34 @@
 
 UPGRADE_README = "https://github.com/vmware/docker-volume-vsphere/blob/master/docs/misc/UpgradeFrom_Pre0.11.1.md"
 
+
+class DBMode(object):
```

Comment:
I think you meant python 2.x. Here's what I got by googling:

Enums have been added to Python 3.4 as described in PEP 435. It has also been backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4 on pypi.

Python Enum can export the name as string naturally, but definitely not as flexible as a user-defined class :)